test/test-add-update-remove.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash 
+pushd "$(dirname $0)/.." > /dev/null 
+PACKAGE_DIR=`pwd` 
+popd > /dev/null
+
+CGS="$PACKAGE_DIR/dist/bin/create-graphql-server.js" 
+INPUT_DIR="$PACKAGE_DIR/test/input" 
+chmod +x $CGS
+ 
+set -e 
+TMPDIR1=`mktemp -d 2>/dev/null || mktemp -d -t 'cgs-test-start'` 
+TMPDIR2=`mktemp -d 2>/dev/null || mktemp -d -t 'cgs-test-end'` 
+
+function finish { 
+  rm -rf $TMPDIR1
+  rm -rf $TMPDIR2 
+  echo 
+  echo 
+  echo "Test failed" 
+} 
+trap finish EXIT
+
+# Prepare two directories for later comparisons with each other
+cd $TMPDIR2 
+JWT_KEY='test-key' $CGS init output-app-start
+cd $TMPDIR1 
+JWT_KEY='test-key' $CGS init output-app-start
+
+cd output-app-start
+$CGS add-type "$INPUT_DIR/Tweet.graphql" 
+$CGS add-type "$INPUT_DIR/User.graphql" 
+$CGS add-type "$INPUT_DIR/Order-before.graphql" 
+# TEST1: Testing of 3 add-types 
+# it (should be pattern User, Tweet, Order in model/index.js)
+# it (should be pattern User, Tweet, Order  in resolvers/index.js)
+# it (should be pattern User, Tweet, Order  in schema/index.js)
+# it (should be a model/User.js, resolver/User.js, schema/User.graphql)
+# it (should be a model/Tweet.js, resolver/Tweet.js, schema/Tweet.graphql)
+# it (should be a model/Order.js, resolver/Order.js, schema/Order.graphql)
+# testing files for existence: 
+if  [ -f "./model/User.js" ] &&
+    [ -f "./model/Tweet.js" ]  &&
+    [ -f "./model/Order.js" ]  &&
+    [ -f "./resolvers/User.js" ]  &&
+    [ -f "./resolvers/Tweet.js" ]  &&
+    [ -f "./resolvers/Order.js" ]  &&
+    [ -f "./schema/User.graphql" ]  &&
+    [ -f "./schema/Tweet.graphql" ]  &&
+    [ -f "./schema/Order.graphql" ]  &&
+    grep -q "User" "./model/index.js"  &&
+    grep -q "Tweet" "./model/index.js" && 
+    grep -q "Order" "./model/index.js" &&  
+    grep -q "User" "./resolvers/index.js" && 
+    grep -q "Tweet" "./resolvers/index.js" && 
+    grep -q "Order" "./resolvers/index.js"  &&
+    grep -q "User" "./schema/index.js" && 
+    grep -q "Tweet" "./schema/index.js" && 
+    grep -q "Order" "./schema/index.js" ; then
+  echo "Test Passed Test1: 3 add-types"
+else 
+  echo "Test Failed Test1: 3 add-types"
+fi
+
+$CGS update-type "$INPUT_DIR/Order-before.graphql" 
+# TEST2: Testing of update-type
+# it (should have a schema/Order.graphql, model/Order.js, resolvers/Order.js file)
+# it (should have a pattern in schema/index.js, resolvers/index.js, model/index.js)
+# testing files for existence: 
+if  [ -f "./model/User.js" ] &&
+    [ -f "./model/Tweet.js" ]  &&
+    [ -f "./model/Order.js" ]  &&
+    [ -f "./resolvers/User.js" ]  &&
+    [ -f "./resolvers/Tweet.js" ]  &&
+    [ -f "./resolvers/Order.js" ]  &&
+    [ -f "./schema/User.graphql" ]  &&
+    [ -f "./schema/Tweet.graphql" ]  &&
+    [ -f "./schema/Order.graphql" ]  &&
+    grep -q "User" "./model/index.js"  &&
+    grep -q "Tweet" "./model/index.js" && 
+    grep -q "Order" "./model/index.js" &&  
+    grep -q "User" "./resolvers/index.js" && 
```

Comment:
Good idea, did it this way:
[ `grep -o "User" "./model/index.js" | wc -l` -eq "1" ]